### PR TITLE
Added Maven Dependencies.

### PR DESCRIPTION
**Pull request by BahubaliPR**
Please do review.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,4 +3,50 @@
...
+
+	<name>utest</name>
+	<url>http://maven.apache.org</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+		<dependency>
```
Comment (gskr442) [inline pom.xml]: What about description for every dependencies you have added?

File: pom.xml [hunk 2]
```diff
@@ -3,4 +3,50 @@
...
+			<version>3.7</version>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/log4j/log4j -->
+		<dependency>
+			<groupId>log4j</groupId>
+			<artifactId>log4j</artifactId>
+			<version>1.2.17</version>
+		</dependency>
+
```
Comment (gskr442) [inline pom.xml]: Remove extra spaces. Use proper code formatting.

File: pom.xml (hunk 1 above)
Comment (BahubaliPR) [reply pom.xml]: Added proper comments for every dependencies.

Comment (gskr442) [reply pom.xml]: Yes I saw that.

File: pom.xml (hunk 2 above)
Comment (BahubaliPR) [reply pom.xml]: Removed extra spaces and formatted properly.

Comment (gskr442) [reply pom.xml]: Got it.

**Pull request merged**